ios/FluentUI.Demo/FluentUI.Demo/Demos/TabBarViewDemoController.swift

```diff
@@ -59,12 +90,57 @@ class TabBarViewDemoController: DemoController {
         NSLayoutConstraint.activate(tabBarViewConstraints!)
 
         tabBarView = updatedTabBarView
+
+        updateBadgeButtons()
+        updateBadgeNumbers()
+    }
+
+    private func updateBadgeNumbers() {
+        if showBadgeNumbers, let tabBarView = tabBarView {
+            let numbers = useHigherBadgeNumbers ? higherBadgeNumbers : badgeNumbers
+
+            tabBarView.setBadgeNumber(numbers[0], for: tabBarView.items[0])
```

Comment:
I think we should be much closer to how native tabbar does. the badge is on the UITabBarItem not through the bar view

Reply:
https://developer.apple.com/documentation/uikit/uitabbaritem/1617065-badgevalue . it could be anything outside of number too.

Reply:
Do we really want to allow other things than numbers in our tab bar? This would probably violate the toolkit designs. I agree that the property should be on the UITabBarItem but I think we should continue to enforce the badge to be a number.

Reply:
Maybe worth talking to whoever made the toolkit designs? It'd surprise me if we purposefully want less support than the default OS control.

Reply:
Updated logic to support any string for the badge value